docassemble/ALAutomatedTestingTests/al_set_up_testing.py

```diff
@@ -218,7 +218,7 @@ def make_pull_request( self ):
 - [x] .github/workflow/run_form_tests.yml
 '''  # TODO: Add issue # if desired
     response = self.repo.create_pull(base=base_name, head=head_name, title=title, body=description)
-    self.pull_url = response.url
+    self.pull_url = response.html_url
```

Comment:
I'm curious. What's the difference between `url` and `html_url`? The documentation I found is very unhelpful: https://pygithub.readthedocs.io/en/latest/github_objects/PullRequest.html

Reply:
I looked at the object returned by the default response in [the github docs on pulls](https://docs.github.com/en/rest/reference/pulls#list-pull-requests).
```js
"url": "https://api.github.com/repos/octocat/Hello-World/pulls/1347",
"html_url": "https://github.com/octocat/Hello-World/pull/1347",
```

Reply:
At the top of that pygithub page it has the link to the github api docs.